### ISSUE TITLE
wide-dataview makes normal text wide as well in live-preview

### ISSUE THREAD

**Issue by HyperEpsilon** [efemkay/obsidian-modular-css-layout#7]
Hi,

I found your wide-dataview to be exactly what I need, however I notice that it makes everything wide in live preview mode. Reading mode looks just fine, but live preview mode is where I spend the majority of my time.

Any ideas on how to fix this? I am currently using the everforest theme.

### Live Preview:
![Obsidian_2022-05-18_22-23-40](https://user-images.githubusercontent.com/6267244/169205112-976bc937-0362-4430-a753-9d80b54058f7.png)

### Reading:
![Obsidian_2022-05-18_22-24-00](https://user-images.githubusercontent.com/6267244/169205115-a78576b7-862c-44fa-8c46-2f4eaeee0de6.png)

**Comment by efemkay** [comment]
yeah. found the issues. will release the fixes in couple of days - wanted to stress test it a bit

**Comment by HyperEpsilon** [comment]
Thanks!

**Comment by efemkay** [comment]
i've made the adjustment. should work now. give it a try n let me know

**Comment by HyperEpsilon** [comment]
Works great!
